### PR TITLE
Update Dangerfile to use master

### PR DESCRIPTION
**Pull request by orta**
I'll see if I can get Danger to raise the error, then fix the error.

Not certain if using yarn this way will compile the TS for Danger, so it might not work at all.

### CODE REVIEW

**Comment by orta** [comment]
Nah, didn't work, will just manually clean it up

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: dangerfile.ts [hunk 1]
```diff
@@ -4,12 +4,11 @@ import { compact, includes, remove } from "lodash"
 
 // For now you can ignore these 3 errors, I'm not sure why
 // they are being raised. They definitely exist in the node runtime.
-import * as fs from "fs"
-import * as os from "os"
-import * as path from "path"
+const fs = require("fs") // tslint:disable-line
+const os = require("os") // tslint:disable-line
+const path = require("path") // tslint:disable-line
```
Comment (orta) [inline dangerfile.ts]: no idea why, but in _this_ project those imports give tsc errors as `import`s works fine in Danger and Peril.

Converting them to `require`s allows tsserver / tslint to continue evaluating.

File: dangerfile.ts [hunk 2]
```diff
@@ -20,16 +19,16 @@ const trivialPR = bodyAndTitle.includes("trivial")
 const acceptedNoTests = bodyAndTitle.includes("skip new tests")
 
 // Custom subsets of known files
-const modifiedAppFiles = modified.filter(path => includes(path, "lib/"))
-const modifiedTestFiles = modified.filter(path => includes(path, "__tests__"))
+const modifiedAppFiles = modified.filter(p => includes(p, "lib/"))
+const modifiedTestFiles = modified.filter(p => includes(p, "__tests__"))
```
Comment (orta) [inline dangerfile.ts]: Swift's order based implicit args would be nice for this kind of stuff.

File: dangerfile.ts [hunk 3]
```diff
@@ -82,8 +81,8 @@ if (testFilesThatDontExist.length > 0) {
 // A component should have a corresponding story reference, so that we're consistent
 // with how the web create their components
 
-const reactComponentForPath = (path) => {
-  const content = fs.readFileSync(path)
+const reactComponentForPath = (filePath) => {
+  const content = fs.readFileSync(filePath).toString()
```
Comment (orta) [inline dangerfile.ts]: This was the crash, was obvious once I made those require  changes.

Comment (alloy) [inline dangerfile.ts]: I usually tell `readFileSync` the encoding as `utf-8`, in which case you get back a string immediately.

Comment (orta) [inline dangerfile.ts]: I did not know this

Comment (alloy) [inline dangerfile.ts]: I believe that even the TS typings pick this up correctly.

**Pull request merged**